Documentation/dev/libvirt-howto.md

```diff
@@ -85,6 +85,35 @@ iptables -I INPUT -p tcp -s 192.168.124.0/24 -d 192.168.124.1 --dport 16509 \
   -j ACCEPT -m comment --comment "Allow insecure libvirt clients"
 ```
 
+If using `firewalld`, simply optain the name of the existing active zone which 
+can be used to integrate the appropriate source and ports to allow connections from
+the IP range used by your cluster nodes. An example is shown below.
+
+````
```

Comment:
The example looks like a shell session, in which case, can you add a prompt and:

    ```console

for highlighting?  That would be:

```console
$ sudo firewall-cmd --get-active-zones
FedoraWorkstation
  interfaces: enp0s25 tun0
```